app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,23 @@
+!!!
+%html
+  %head
+    %title Instagram
+    %link{:rel => :stylesheet, :type => :"text/css", :href => "/assets/layout"}
+    = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+
+  %body
+    %div{:class => "navbar"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace